transport_nantes/topicblog/views.py

```diff
@@ -372,12 +384,18 @@ def _create_email_object(self, tb_object, context: dict,
                              recipient_list: list, send_record,
                              from_email: str = settings.DEFAULT_FROM_EMAIL) \
             -> mail.EmailMultiAlternatives:
-        """
-        To send emails to multiple persons, django send_mail function isn't
-        enough, we need to create a EmailMultiAlternatives object to be able,
-        for example to put recipients in BCC field or add attachments.
-        It also improves the performances by reusing the same connexion to SMTP
-        server.
+        """Create an EmailMultiAlternatives object.
+
+        To send emails to multiple persons, django send_mail function
+        isn't enough, we need to create a EmailMultiAlternatives
+        object to be able, for example to put recipients in BCC field
+        or add attachments.  It also improves the performances by
+        reusing the same connexion to SMTP server.  **BUG 1** We don't
```

Comment:
BUG1 : The docstringn is outdated in the sense we don't use SMTP anyway, so better delete it to not maintain confusion.